helm-chart/renku-notebooks/values.yaml

```diff
@@ -22,8 +22,8 @@ gitlab:
 replicaCount: 1
 
 image:
-  repository: renku/renku-notebooks
-  tag: 'latest'
+  repository: rrrrrok/renku-notebooks
```

Comment:
I guess this was committed by accident. See also below...

Reply:
Yep I saw, I'll fix it

Reply:
👍